content/docs/user-guide/what-is-dvc.md

```diff
@@ -1,16 +1,18 @@
 # What Is DVC?
 
-**Data Version Control** is a new type of data versioning, workflow and
-experiment management software, that builds upon [Git](https://git-scm.com/)
-(although it can work stand-alone). DVC reduces the gap between established
-engineering tool sets and data science needs, allowing users to take advantage
-of new [features](#core-features) while reusing existing skills and intuition.
+**Data Version Control** is a new type of data versioning, workflow and machine
+learning experiment management software that builds upon
+[Git](https://git-scm.com/) (although it can work stand-alone). DVC reduces the
+gap between established engineering tool sets and data science needs, allowing
+users to take advantage of new [features](#core-features) while reusing existing
+skills and intuition.
 
-![](/img/reproducibility.png) _DVC codifies data and ML experiments_
+![](/img/reproducibility.png) _DVC codifies data and machine learning
+experiments_
```

Comment:
I wonder whether having "ML" along with "machine learning" in the same doc also helps, or should abbreviated terms be avoided?

Reply:
> I wonder whether having "ML" along with "machine learning" in the same doc also helps, or should abbreviated terms be avoided?

Yes, it does help. Ideally we have both, including the combined usage. For example, "ML experiments" and "machine learning experiments".

Reply:
OK! So I think this change can be revered? Since the full term will be added somewhere before this image (probably). An img caption seems like a good place for the abbr form.

Reply:
Agree abbr better for caption and added full usage in rewritten paragraph.